Corefile.example

```diff
@@ -1,14 +1,32 @@
 . {
-   debug
-   netbox {
-     token  0123456789abcdef0123456789abcdef01234567
-     url    http://localhost:8000/api/ipam/ip-addresses
-     localCacheDuration 1h
-   }
+  debug
+  netbox {
+    token 0123456789abcdef0123456789abcdef01234567
+    url http://localhost:8000/api/ipam/ip-addresses
+    localCacheDuration 1h
+  }
 
-   hosts {
-        10.0.0.1 example.org
-	10.0.0.3 definitely-not-in-netbox.org
-        fallthrough
-   }
```

Comment:
So, now hosts which are not found in netbox will be looked in the inline hosts section?

Reply:
I have added some comments relating to this, but in a word? No.

This all depends on the ordering of the plugins in `plugins.cfg`.

If *netbox* was **before** *hosts* in `plugins.cfg` then *netbox* will handle all queries and never pass the query to *hosts* as there is no `fallthrough` directive in the *netbox* config block.

If *netbox* was **after** *hosts* in `plugins.cfg` then *hosts* will handle the query first and then pass the request to *netbox* if it cannot handle it due to the `fallthrough` directive in the *hosts* config block.